.eslintrc.js

```diff
@@ -52,13 +52,14 @@ module.exports = {
             alphabetize: { order: 'asc', caseInsensitive: true },
           },
         ],
-        'import/no-named-as-default-member': 'off',
-        '@typescript-eslint/no-unused-vars': ['error'],
+        "import/no-named-as-default": 'off',
+        "@typescript-eslint/no-unused-vars": ["warn", { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }],
```

Comment:
The idea here is to ignore the unused vars rule for variable names `_likeThis`

Reply:
curious, which case are you covering with this change? 🤔 I believe we should remain the error for all unused variables

Reply:
I think it's useful during development to declare a variable that maybe you don’t use right away and it still compiles with a warning.

The _ is to remove that warning on some specific cases:
- unused function arguments.
```
function fn(_first, second) { ... }
```
- for destructuring an array:
```
const [_first, second] = someArray;
```
- [here](https://github.com/dbrosio3/blackmarket-frontend/blob/f7281ff20be69ed52c29dc2a10a06347c92c7166/src/features/register/Register.tsx#L12), for example, I plan on using the `response` but I don’t feel like defining the necessary types to pass it as an argument of `onRegister` yet. 

This last case is a bit of a stretch, I'll remove that variable.
(The warning was a bit annoying so I intentionally ignored it 🙈)

Reply:
I believe the general rule should be to have an error and then disable the specific cases where you want to ignore it